src/memory.h

```diff
@@ -146,6 +151,9 @@ class MutableMemory : public Memory {
       TRITONSERVER_MemoryType* memory_type = nullptr,
       int64_t* memory_type_id = nullptr);
 
+  // Set the memory to the specified value.
+  Status SetMemory(int value);
```

Comment:
small nit: while the value of the underlying apis use an int but convert to unsigned char - maybe we define this api using an unsigned char to be more precise -

Reply:
fixed.